python/paddle/v2/fluid/backward.py

```diff
@@ -199,12 +199,76 @@ def _op_can_be_removed_(op_desc, no_grad_set):
     return op_descs
 
 
+import proto.framework_pb2 as framework_pb2
+
+
+def serialize_op_decs(op_desc):
+    protostr = op_desc.serialize_to_string()
+    proto = framework_pb2.OpDesc.FromString(str(protostr))
+    return proto.__str__()
+
+
+def _callback_lookup_(op):
+    """
+    Only used in _append_backward_ops_
+    Build and returns a callback function for certain op. For example
```

Comment:
Can you add more comment about what is callback function? (e.g, is it something gets called after a OP is completed?)